libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/store/EventStoreManager.java

```diff
@@ -238,9 +238,6 @@ private String encrypt(String data) {
     }
 
     private String decrypt(String data) {
-        if (encryptionKey.getBytes().length == 24) {
```

Comment:
We don't need this anymore because there is no `PaperDB` prior to `8.2`, so this will never get called with the old code.